header/headertest/testing.go

```diff
@@ -349,10 +349,11 @@ func ExtendedHeaderFromEDS(t *testing.T, height uint64, eds *rsmt2d.ExtendedData
 
 func CreateFraudExtHeader(
 	t *testing.T,
+	odsSize int,
```

Comment:
can't you just take this from len(eh.DAH.rowroots)

Reply:
Yes. Thank you